centralized-sampling-tests/sample-apps/golang-http-server/application.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"strconv"
+	"time"
+
+	"go.opentelemetry.io/contrib/propagators/aws/xray"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc"
+	sample "go.opentelemetry.io/contrib/samplers/aws/xray"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/sdk/trace"
+	oteltrace "go.opentelemetry.io/otel/trace"
+
+	"google.golang.org/grpc"
+)
+
+func getSampledSpanCount(name string, totalSpans string, attributes []attribute.KeyValue) int {
+	tracer := otel.Tracer(name)
+
+	var sampleCount = 0
+	totalSamples, _ := strconv.Atoi(totalSpans)
```

Comment:
```suggestion
	totalSamples, _ := strconv.Atoi(totalSpans)
```
Is this returning an error? Do not ignore it.